doc/specs/drafts/#3327 - Application Theming/#3327 - Application Theming.md

```diff
@@ -373,7 +373,7 @@ changes, or the active pane in a tab changes:
       `TabRowControl` to match.
 
 The `tab.cornerRadius` might be a bit trickier to implement. Currently, there's
-not a XAML resource that controls this, nor is this something that's exposed by
+no XAML resource that controls this, nor is this something that's exposed by

```

Comment:
I didn't leave it enabled, but there's a pattern that flags `not`/`nor` suggesting `neither`/`nor`. I believe in this case `no` is better.